storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -29,6 +29,44 @@ In future {product-title} versions, volumes provisioned using existing in-tree p
 After full migration, in-tree plug-ins will eventually be removed in future versions of {product-title}.
 ====
 
+=== Storage policy
+
+The vSphere CSI Operator Driver storage class now uses vSphere's storage policy. {product-title} automatically creates a storage policy that targets datastore configured in cloud configuration:
+[output, yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: thin-csi
+provisioner: csi.vsphere.vmware.com
+parameters:
+  StoragePolicyName: "${STORAGE_POLICY_NAME}"
+volumeBindingMode: WaitForFirstConsumer
+# Volume expansion is disabled here because volume expansion in the vSphere
+# CSI driver is still in beta and has several limitations. (see
+# https://vsphere-csi-driver.sigs.k8s.io/supported_features_matrix.html).
+# It is not supported for all datastores (vSAN only right now) and there is
+# not yet an e2e test for it.
+allowVolumeExpansion: false
+reclaimPolicy: Delete
+----
+
+=== Version requirements
+
+The vSphere CSI Operator Driver requires certain minimum versions for products that make up the virtual environment as shown in the following table.
+
+.Version requirements for vSphere CSI Operator Driver
+[cols=", ^v, width="100%", options="header"]
+|===
+|Virtual environment product |Required version
+
+|VM hardware version | 15 or later
+|vSphere ESXi hosts | 6.7u3 or later
+|vCenter host   | 6.7u3 or later
+|===
+
+If you are using earlier versions, you will receive alert notifications regarding this. Also, you may not be able to upgrade to {product-title} 4.10 unless you are compliant.
```

Comment:
Not sure if the first sentence is required. We already say that certain versions are required in the docs. If you go < those versions, you should expect to see warnings. 

```suggestion
You may not be able to upgrade to {product-title} 4.10 unless you are compliant.
```

I'm not sure about this. Does it mean that I might be able to brute force my way through? Does it mean that it's a coin flip? Does it mean that some configurations work and some don't? Does it mean that upgrades are not supported?

If it's that last one, IMO, just say it.

Reply:
After latest meeting with PMs, we are not going to block upgrades from 4.9 to 4.10 anymore. It will be done in 4.10 to 4.11 now.